scripts/eqtl_hail_batch/calculate_ld.py

```diff
@@ -0,0 +1,64 @@
+"""Read in correlation results and claculate ld values"""
+
+import hail as hl
+import click
+import pandas as pd
+
+
+@click.command()
+@click.option('--input-path', help='A path prefix of where to input files are located')
+def query(input_path):
+    """calculate LD values"""
+
+    hl.init(default_reference='GRCh38')
+
+    tmp_dir = input_path.replace(input_path.split('/')[2], input_path.split('/')[2] + '-tmp')
+    mt_path = f'{tmp_dir}/genotype_table.mt/'
+    mt = hl.read_matrix_table(mt_path)
+    mt = mt.annotate_rows(
+            global_bp=hl.locus(
+                mt.locus.contig, mt.locus.position
+            ).global_position(),
+        )
+    # if running on entire genome, concatenate all significant
+    # correlation matrices together, then read in as one df
+    # example: https://stackoverflow.com/questions/20906474/import-multiple-csv-files-into-pandas-and-concatenate-into-one-dataframe
+    significant_snps_path = f'{input_path}correlation_results.csv'
+    significant_snps_df = pd.read_csv(
+        significant_snps_path, sep=' ', skipinitialspace=True
+    )
+    t = hl.Table.from_pandas(significant_snps_df)
+    # only keep rows whose FDR is < 0.05
+    # t = t.filter(t.fdr < 0.05)
```

Comment:
Should this be uncommented? Alternatively, if it's no longer needed because you only load significant SNPs into the dataframe above, can you expand the significant SNPs comment to include this threshold information (FDR < 0.05) before removing the unnecessary lines.

Reply:
You're 100% correct- this should be uncommented (I think that's a holdover from testing).